content/en/seachat/seachat-manual/03-add-knowledge/04-template-file-upload.md

```diff
@@ -0,0 +1,132 @@
+---
+title: "Template Upload"
+description: "Template Upload Using CSV and JSON Files"
+lead: ""
+date: 2024-04-11 10:43:51.069 +0100
+lastmod: 2024-04-11 10:43:51.069 +0100
+weight: 33
+draft: false
+images: []
+aliases:
+  - /en/seachat/seachat-intro/03-add-knowledge/template-upload/
+---
+
+## Overview
+SeaChat provides four methods to upload files to your agent. We will focus on the **Upload Spreadsheet** method in this tutorial and by the end of the tutorial your SeaChat agent will have a customized knowledge base at your service.
```

Comment:
it's better not to mention "four". We will keep adding and you don't want to constantly update blogs